scripts/tasks/ts.js

```diff
@@ -2,11 +2,18 @@
 
 const path = require('path');
 const { tscTask, argv } = require('just-scripts');
+const { readdirSync } = require('fs');
```

Comment:
why is this added?

Reply:
Left overs from the first commit. You can see I was programmatically trying to enumerate all the @types except Jasmine and pass it in the commandline to tsc